dwr-hecdssvue/build.gradle

```diff
@@ -0,0 +1,81 @@
+plugins{
+    id 'library.deps-conventions'
+    id 'library.java-conventions'
+
+    // The following plugin is used to build the OSGI bundle
+    id 'biz.aQute.bnd.builder' version '6.4.0'
+}
+
+configurations {
+    eclipseLibs
+    implementation.extendsFrom(eclipseLibs)
+}
+
+dependencies {
+    //additional dependencies
+    implementation libs.hec.monolith
+    implementation libs.hec.nucleus.metadata
+    implementation libs.dssgui
+
+    //module dependencies
+    implementation project(':third-party')
+    implementation project(':wrims-ide')
+
+    //eclipse libs
+    implementation fileTree(dir: '../eclipse-luna-libs', include: ['*.jar'])
+
+    //add test dependency on junit
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.1'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
+    testRuntimeOnly fileTree(dir: '../eclipse-luna-libs', include: ['*.jar'])
+}
+
+configurations.configureEach {
+    exclude group: "javax.media", module: "jai_core"
+    exclude group: "javax.media", module: "jai_codec"
+    exclude group: "com.sun.media", module: "jai_imageio"
+}
+
+test {
+    useJUnitPlatform()
+    include '**/OSGiModuleTest.class'
```

Comment:
This path is confusing me, I haven't seen an include of a class file before and I would expect it to be at least constrained to the build directory.

Reply:
This test was removed.